packages/bot/src/common/behaviors/dice/dice.behavior.base.ts

```diff
@@ -146,6 +153,40 @@ export abstract class DiceBehaviorBase extends BehaviorBase {
     return null;
   }
 
+
+
+  protected rollSpoileredV1(trigger: Trigger, logTag: string, content: string, context: BehaviorContext, requireDice: boolean): string[] | null {
+    const match = content.match(/([sS]#)(?:(\d+)#\s*)?(.*)/i);
+
+    if (!match) {
+      return null;
+    }
+    const diceWithoutSpoiler = match[3] ? match[3] : content;
+
+
+    const lines: string[] = [];
+    const result = this.parsers.v1.tryParse(diceWithoutSpoiler);
+    if (!result) { return null; }
+
+    // don't be too aggressive with the replies
+    const hasEnoughDice = requireDice ? result.dice > 0 : true;
+    const hasPrefix = isKnownPrefix(context.messageConfiguredOptions?.prefixStyle);
+    const shouldReply = hasPrefix || (result.depth > 1 && hasEnoughDice);
+    if (!shouldReply) { return null; }
+
+    const response = this.buildMessage(result, requireDice);
+
+    if (response && shouldReply) {
+      lines.push("||" + response + "||");
```

Comment:
Note for me later: See if this has interactions with formatting

Reply:
It did. Resolving this well would require passing another arg to the `buildMessage` function to determine if we escape `|` characters.

![image](https://github.com/user-attachments/assets/5d1d9d67-1708-4130-87e2-7e0669166d49)

Reply:
https://github.com/rollem-discord/rollem-discord/issues/229